requirements.txt

```diff
@@ -2,3 +2,4 @@ boto3==1.16.40
 kubernetes==12.0.1
 PyYAML==5.4
 pytest-xdist==2.2.0
+Jinja2==2.11.3
```

Comment:
This is not the place for this. These requirements are for the `acktest` source package that all repositories import as part of their integration tests. I would add a new `requirements.txt` in `prow/jobs/` alongside the Python files, there.

Reply:
Although I think moving the prow jobs to a new repository altogether would be a better bet. But for now here is fine.

Reply:
I will cleanup and separate the `requirements.txt` for now.